mesonbuild/rewriter.py

```diff
@@ -709,6 +709,87 @@ def find_node(src):
                 if root not in self.modified_nodes:
                     self.modified_nodes += [root]
 
+        if cmd['operation'] == 'extra_files_add':
```

Comment:
This doesn't really matter, but for consistency, this should also be an `elif`.

Reply:
Great catch, didn't notice when I copy-pasted the sources add/rm logic. Will fix.